requirements.txt

```diff
@@ -44,6 +44,10 @@ git+https://github.com/CenterForOpenScience/modular-odm.git@develop
 # Python markdown extensions for comment emails
 git+git://github.com/aleray/mdx_del_ins.git
 
+# Kombu with the ability to specify queue priority
```

Comment:
It's the same version of Kombu that we were using, with one additional commit cherry-picked in: https://github.com/celery/kombu/commit/c20f85446926cf0cfdadaa196113835282a33612

It seemed not to break anything for me, and I used both the default `amqp` and `rabbitmqctl` to verify these queues were being created with the proper priority levels.